pkg/virt-config/feature-gates.go

```diff
@@ -89,3 +90,7 @@ func (config *ClusterConfig) SriovEnabled() bool {
 func (config *ClusterConfig) HostDiskEnabled() bool {
 	return config.isFeatureGateEnabled(HostDiskGate)
 }
+
+func (config *ClusterConfig) MacvtapEnabled() bool {
```

Comment:
I see the there is also [NetworkConfiguration](https://github.com/kubevirt/kubevirt/blob/1b1279dea2f149ba446ef2b8b7fe51f2526f842c/staging/src/kubevirt.io/client-go/api/v1/types.go#L1338) the `PermitSlirpInterface` and `PermitBridgeInterfaceOnPodNetwork` are there.

Why did you decide to put the macvtap feature gate here and not there?

Reply:
I think the NetworkConfiguration makes sense since those 2 things - Bridge binding mechanism & Slirp binding mechanism are entirely implemented within KubeVirt.

Macvtap binding mechanism, on the other hand, is partly implemented outside of the KubeVirt repository.

Do you foresee any issue , or know of any advantage of using the NetworkConfiguration - instead of the feature fates ?

As I see it, having macvtap also on the `NetworkConfiguration` would be redundant.